Web/Utility/Rest.cs

```diff
@@ -733,7 +733,7 @@ public static class Rest
         /// </param>
         /// <param name="abortToken">Optional cancellation token.</param>
         /// <returns>The finished unity web request. Can be NULL if operation cancelled or error occurs.</returns>
-        private static async Task<WebRequestResponse> RunGetAsync(
+        private static async Task<WebRequestResponseWithHandler> RunGetAsync(
```

Comment:
Revert to `WebRequestResponse`. This one is deceptive because of the naming. This method does take a file download handler so that the data is written directly to disc, which is memory-efficient for Unity compared to a standard download handler. However, this all happens and is disposed of after the request succeeds. With `DownloadFileHandler`, you can't directly access the bytes, which results in access exceptions. Therefore, the return type does not require a handler to be associated with it.